content/en/docs/admin/admission-controllers.md

```diff
@@ -60,13 +61,12 @@ admission control plugins:
 kube-apiserver --enable-admission-plugins=NamespaceLifecyle,LimitRanger ...
 ```
 
-{{< note >}}
 **Note**: Depending on the way your Kubernetes cluster is deployed and how the
 API server is started, you may need to apply the settings in different ways.
 For example, you may have to modify the systemd unit file if the API server is
 deployed as a systemd service, you may modify the manifest file for the API
 server if Kubernetes is deployed in a self-hosted way.
-{{< /note >}}
```

Comment:
These need to be reverted to the red lines 63 and 69, to use the Hugo shortcodes.